packages/views/screens/managers/hwWalletLogin/selectAccount/index.js

```diff
@@ -2,11 +2,11 @@
 import { connect } from 'react-redux';
 import { settingsUpdated } from 'src/modules/settings/store/actions';
 import { login } from '@auth/store/action';
-import { getActiveTokenAccount } from '@wallet/utils/account';
+import { selectActiveTokenAccount } from '@common/store';
```

Comment:
`selectActiveTokenAccount` is related to the account, could you move it back to the account domain?

Reply:
There's a long discussion about this. We can talk if you're interested.